### DISCUSSION TITLE
Can we have checkboxes to select multiple files in Nemo?

### DISCUSSION THREAD

**Discussion by archisman-panigrahi** [linuxmint/discussions#353] [Ideas]
The Windows file explorer and the Dolphin file manager provide an option to select multiple files using checkboxes, just using mouse, without pressing `ctrl` in the keyboard. This is extremely handy.

It would be great to have this feature in nemo.

**Comment by Jopp-gh** [comment]
Just drag a selection with your mouse. That's very quick and handy. 

Ofc if you need to subtract files, you'll need a keyboard too

**Reply by archisman-panigrahi** [reply]
I know that dragging works. But if I want to select two files which are far apart, I will need a keyboard. This is why I am proposing we add the checkbox feature

**Comment by poyribn** [comment]
That would finally make "single&nbsp;Click to open&nbsp;Files" useful.

On Microsoft&nbsp;Windows, i am used&nbsp;to
clicking the Checkboxes to select Files
and clicking the Files themselves to open them.
None of that double&#x2011;Click&nbsp;Nonsense.

**Comment by archisman-panigrahi** [comment]
I sent a pull request with this feature in https://github.com/linuxmint/nemo/pull/3811. Please test and give feedback
